tools/devops/provision-objective-sharpie.csx

```diff
@@ -0,0 +1,32 @@
+#load "provision-shared.csx"
+
+// Provision Objective Sharpie
```

Comment:
what's calling this file ?

Reply:
There is a better way to approach this, I'll add it, we can use make to get the sharpie version and add it to the provisioning